jsonschema_extractor/typing_extractor.py

```diff
@@ -109,14 +97,10 @@ def _is_sequence(typ):
     # PEP_560 deprecates issubclass for
     # List types, for the time being
     # we'll support a specific escape hatch.
-    if PEP_560:  # pragma: no cover
-        return isinstance(typ, _GenericAlias) and typ.__origin__ is list
-    else:  # pragma: no cover
-        return issubclass(typ, Sequence)
+    if hasattr(typ, "__origin__"):
+        return issubclass(typ.__origin__, Sequence)
```

Comment:
One note - putting this (and the below check) inside the conditional means that, if `typ` does _not_ have `__origin__`, the code does not have an explicit `return` for all branches. Looking at how the results are used, this may not be an issue, but can be a violation of some style guides

Reply:
yep! good call. I think regardless of style guide it's less error-prone.